person.rb

```diff
@@ -1,5 +1,8 @@
-class Person
+require_relative 'nameable'
```

Comment:
- [ ] Good job so far, :clap: , however, I can see that you have not followed the project requirement, according to the project requirement, you are supposed to create a `Nameable` class, `BaseDecorator` class, `CapitalizeDecorator` class,  `TrimmerDecorator` class and their properties. please follow the requirements given [here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md#prepare-capitalizedecorator-and-trimmerdecorator) to implement your task.